tests/report/report_plugin_test.py

```diff
@@ -82,7 +82,7 @@ def test_subtests(subtests):
         extra_args
     )
 
-    result.assert_outcomes(passed=2, failed=3, errors=1, xfailed=1, xpassed=1, skipped=1)
+    result.assert_outcomes(passed=2, failed=1, errors=1, xfailed=1, xpassed=1, skipped=1)
```

Comment:
What this plugin is doing so the number of failures is less than before? 
If I have some tests written in Python and those tests are using subtests, all subtests failures will be hidden, will not? 
It sounds like something dangerous, like "make all tests green".